docs/features/secrets/general.md

```diff
@@ -60,4 +60,13 @@ After a hard refresh you can use the latest secret again and proceed your work.
 Secret secret = await cachedSecretProvider.GetSecretAsync("EventGrid-AuthKey", ignoreCache: true);
 ```
 
+## Invalidates a secret from the cach
+In some scenarios you'd like to remove the entry from the cache, so the next the secret is retrieved, a new lookup will be done in the provider.
```

Comment:
How about this?

```suggestion
In some scenarios you'd like to remove a cache entry so that the secret will be retrieved from the provider when a new lookup will be done.
```